lib/schema-converters.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { iter } = require('metarhia-common');
+const { all: decorators } = require('./decorators');
+
+const defaultOptions = {
+  useEsm: false,
+};
```

Comment:
Maybe to a single line

Reply:
This is the style we have for all other such options/defaultOptions and don't want to have it differently here.

Reply:
Ok